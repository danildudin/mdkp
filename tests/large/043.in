50 16 0
74 649 114 18 383 641 921 473 884 565 447 653 692 507 114 831 824 682 687 102 55 801 969 266 953 689 564 891 789 187 916 564 224 195 2 327 847 977 461 216 893 107 737 773 589 555 460 869 724 402 
28 260 800 462 495 448 110 288 904 33 899 179 473 138 19 212 549 359 885 352 127 564 575 217 658 930 825 387 923 962 313 983 670 277 107 460 504 482 716 597 575 336 679 553 132 464 645 635 116 999 
605 133 173 895 30 627 277 121 418 191 258 741 412 976 642 394 538 243 206 282 718 252 102 820 320 686 207 142 876 166 373 887 49 418 105 862 801 442 269 687 357 1 746 831 973 95 80 52 534 717 
291 314 392 527 534 402 64 918 121 744 824 996 58 550 126 538 959 518 645 31 549 438 526 958 264 111 8 795 920 691 626 931 421 391 478 470 355 418 554 72 116 105 629 386 370 791 914 363 882 686 
805 787 572 47 378 52 180 512 786 482 32 862 556 644 364 686 230 117 442 67 53 940 778 355 867 193 164 256 209 931 691 470 696 964 677 495 644 294 535 775 86 888 767 367 547 401 976 959 950 869 
465 65 505 481 990 142 555 757 844 869 581 54 587 620 33 688 947 252 81 331 461 648 344 291 155 568 177 247 12 692 514 73 176 652 440 654 377 471 188 109 255 451 188 323 721 494 539 49 85 243 
664 712 123 595 623 591 389 583 694 328 539 970 244 212 715 689 391 5 78 548 329 679 464 925 578 78 312 379 175 526 636 767 629 958 35 820 308 978 981 448 177 422 934 355 103 709 295 967 103 943 
680 638 667 437 116 887 555 163 252 774 495 965 944 808 650 690 299 41 505 634 62 458 843 702 177 784 655 87 489 511 96 453 346 849 993 934 246 609 534 25 724 511 74 210 520 890 762 327 104 995 
562 590 682 860 851 360 977 929 244 105 590 440 208 777 82 313 957 910 180 960 368 600 423 409 933 332 353 700 7 299 941 209 359 708 245 448 806 593 390 35 58 69 976 101 514 507 122 225 513 256 
637 883 257 500 455 662 117 79 280 727 363 434 578 111 114 468 315 535 431 820 878 891 969 647 816 556 981 404 48 807 167 210 897 445 814 486 988 552 406 795 236 852 229 570 322 394 281 599 580 818 
898 13 245 824 637 708 610 323 858 572 582 632 796 646 401 824 193 624 183 627 846 522 325 666 747 831 553 752 731 528 494 481 125 712 131 405 411 721 885 747 692 52 983 3 359 923 529 1000 458 521 
591 25 945 554 666 889 292 176 661 268 932 322 363 149 802 985 706 475 539 579 212 176 345 756 891 674 499 611 852 222 700 305 692 274 483 790 471 361 512 711 865 380 356 565 948 222 64 677 513 792 
459 489 140 973 905 732 995 710 546 913 419 799 10 756 59 948 483 983 174 173 95 751 607 698 906 988 94 866 63 849 218 626 289 366 614 677 571 514 38 544 585 783 183 972 459 969 835 551 865 946 
657 320 72 152 139 588 505 648 498 33 139 883 128 137 809 552 703 319 942 2 784 953 967 380 70 652 621 366 987 713 929 823 151 183 876 347 281 548 280 772 961 966 415 896 994 773 972 815 915 103 
139 683 20 828 294 7 626 566 665 385 832 359 411 416 9 106 782 426 24 901 418 124 906 522 1000 562 142 441 740 765 522 184 947 573 869 947 638 581 700 192 24 617 799 502 803 660 539 597 485 334 
981 528 667 376 703 385 379 449 113 521 645 14 603 893 218 679 928 115 388 938 273 557 82 106 962 284 933 156 784 592 55 294 218 814 616 410 145 947 453 648 639 773 315 672 968 623 384 266 736 735 
854 925 86 263 402 62 222 60 945 682 768 943 910 418 586 499 170 162 88 147 290 92 934 761 158 396 873 420 379 1000 559 531 266 587 849 47 236 226 511 31 978 319 639 123 63 106 745 364 887 623 
42706 2704 86440 70827 8854 89505 85009 42588 51836 35980 46455 64488 83035 24524 99774 1334 
