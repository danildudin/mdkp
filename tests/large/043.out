6873
9
6 7 16 17 18 21 24 37 45 
